package.json

```diff
@@ -652,6 +652,13 @@
           "name": "Runs",
           "contextualTitle": "Terraform Cloud runs"
         }
+      ],
+      "terraform-cloud-panel": [
+        {
+          "id": "terraform.cloud.run.plan",
+          "name": "Plan",
+          "contextualTitle": "Terraform Plan"
```

Comment:
```suggestion
```

A `contextualTitle` is shown whenever a view isn't inside a treeview, which this will never be by default, so this view will always use the contextualTitle. This results in `TERRAFORM PLAN: PLAN` as the effective title, which in addition to being redundant is confusing as we're asking TFC.

I suggest removing this, as VS Code by default uses the viewContainer/panel title and view name, which will result in `TERRAFORM CLOUD: PLAN`